hollow-jar/web/pom.xml

```diff
@@ -23,16 +23,12 @@
             <scope>provided</scope>
```

Comment:
Please remove this dependency as well, should be covered by the `jboss-javaee-web-7.0` uberjar.